### PR TITLE
Scan workflows

### PR DESCRIPTION
**Pull request by jsiever1**
Add a scan for Reusable Workflows.
The workflows will be added in a new separated field 'workflows' of the json output file
The name is the name listed in the workflow or when not available the name of the workflow yml file (without the yml )
```
    {
	
	...
  ],
  "workflows": [
    {
      "name": "Deploy",
      "repo": "dolly",
      "isArchived": false,
      "downloadUrl": "...../.github/workflows/deploy.yml"
    },
    {
      "name": "called workflow",
      "repo": "Sample_Repo_2",
      "isArchived": false,
      "downloadUrl": "...../.github/workflows/Callee.yml"
    },

	....

]
```

### CODE REVIEW

**Comment by rajbos** [comment]
Great work @jsiever1! Could you add a test in [pr_validation.yml](https://github.com/devops-actions/load-available-actions/blob/main/.github/workflows/pr_validation.yml) that validates the reusable workflow list is not empty?

**Comment by jsiever1** [comment]
Added the test for the workflows
image

**Comment by jsiever1** [comment]
Readme Updated with New workflow scan and fixes of markdown (following the markdown linter)

File: README.md [hunk 1]
```diff
@@ -2,20 +2,26 @@
...
+
 - action.yml
 - action.yaml
 - Dockerfile
 - dockerfile
+- .github/workflows/<workflow file>.yml
+
 For the Dockerfiles we search for the required labels to identify them as actions.
 
+For the reusable workflow a search is done of the a workflow file contains '`workflow_call:`'
```
Comment (rajbos) [inline README.md:19]: Typo:
```suggestion
For the reusable workflow a search is done if the workflow file contains '`workflow_call:`'
```

File: README.md [hunk 2]
```diff
@@ -47,16 +57,26 @@ Repo permissions needed for the access token:
...
+      }
     ]
 }
 ```
+
 Properties:
 |Name|Description|
 |----|-----------|
 |lastUpdated|The date and time this action list was created. Format = YYYYMMDD_HHmm|
 |actions|The list of actions available in the workflows in the organization.|
```
Comment (rajbos) [inline README.md:77]: Add the info here as well:
```suggestion
|workflows|The list of reusable workflows availabe in the repositories in the organization.|
```

**Review by rajbos — COMMENTED** [review]
2 more suggestions

**Review by rajbos — APPROVED** [review]

**Pull request merged**